test-suite-freemarker-fieldset/src/test/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+micronaut.views.freemarker.default-extension=html
+micronaut.views.freemarker.incompatible-improvements=2.3.32
```

Comment:
I needed to add this so that Apache Freemarker had visibility to Java default methods